plugins/ingest/src/main/java/org/elasticsearch/ingest/Pipeline.java

```diff
@@ -22,10 +22,7 @@
 
 import org.elasticsearch.ingest.processor.Processor;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
this is what the `CONTRIBUTIN.md` says:

```
Don't worry too much about imports. Try not to change the order but don't worry about fighting your IDE to stop it from switching from * imports to specific imports or from specific to * imports.
```

So it is lenient and up to the IDE or editor being used.